android/mwcore/src/main/AndroidManifest.xml

```diff
@@ -16,14 +16,15 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme.NoActionBar">
         <activity
-            android:name=".ui.login.LoginActivity"
+            android:name="org.smartregister.fhircore.engine.ui.appsetting.AppSettingActivity"
```

Comment:
@KhumboLihonga - will this not set the launch streen AppSetting? Do we want that?

Reply:
Ideally, no we don't. This is how the app is now configured in the engine so this PR was just about fixing it to match that and get the app building again. I can submit another PR that removes that functionality.

Reply:
I ran the app and realized that they have standardized it that way. Is this going to be a one time thing when you ran the app for the first time? Or you have to write it the app id everytime you want to log in?

Reply:
I'm not completely sure, I need to look into it more. Like I said, the focus was just about getting the app to build because there were quite a number of breaking changes from the time we forked the repo.

Reply:
I think it might have something to do with the configurability of the application and having the setting screen be the first one to appear might allow for some configurations to hapen before logging in am assuming something like if its ADDO or HF (Am using afyatek as an example here) then the app can fetch questionnaires based on confugurations before logging in